examples/run_multiple_choice.py

```diff
@@ -362,7 +362,7 @@ def main():
                         help="Whether to run eval on the dev set.")
     parser.add_argument("--do_test", action='store_true', help='Whether to run test on the test set')
     parser.add_argument("--evaluate_during_training", action='store_true',
-                        help="Rul evaluation during training at each logging step.")
+                        help="Run evaluation during training at each logging step.")
```

Comment:
Could you please help to fix a `--overwrite_cache ` problem at line 296 of this file in this PR.
I think `line 296` should be changed to 
```
if os.path.exists(cached_features_file) and not args.overwrite_cache:
```
to make `--overwrite_cache` valid.
Since I cannot add a comment on line 296, I put it in the last change of this file. Thank you.